bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/UL_Radion_WW_inclu/UL_Radion_WW_inclu_narrow_M/UL_Radion_WW_inclu_narrow_M_run_card.dat

```diff
@@ -0,0 +1,272 @@
+#*********************************************************************
+#                       MadGraph5_aMC@NLO                            *
+#                                                                    *
+#                     run_card.dat MadEvent                          *
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#*********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run 
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  .false.     = gridpack  !True = setting up the grid pack
```

Comment:
why?

Reply:
Similar to the previous comment, this is a remnant from building the new card over the older [reference](https://github.com/cms-sw/genproductions/blob/de84144b2fa4ad3ce1d9960af7b791af8e45a831/bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/exo_diboson/Spin-0/Radion_WW_inclu_narrow/Radion_WW_inclu_narrow_M/Radion_WW_inclu_narrow_M_run_card.dat#L26). I am unsure of the motivation behind this line.